tools/single_use/instrument_double_escape_report.php

```diff
@@ -53,6 +53,15 @@
     $useObjects =true;
```

Comment:
Updating this tool cause it still might be used in this release by some projects and it was broken